polaris-for-vscode/package.json

```diff
@@ -42,9 +42,6 @@
     "vscode-languageserver-textdocument": "^1.0.4"
   },
   "devDependencies": {
-    "@rollup/plugin-babel": "^5.3.1",
-    "@rollup/plugin-commonjs": "^21.1.0",
-    "@rollup/plugin-node-resolve": "^13.2.1",
```

Comment:
It's strange that these are erroring here and not in `polaris-tokens`:
![image](https://user-images.githubusercontent.com/32409546/170763711-fb8efe9a-5113-47f8-a9e8-c1331d1a80da.png)

Perhaps, the build is just failing first in the `polaris-for-vscode` `rollup.config.js`. To quickly unblock this PR we could inline disable the eslint rule for these imports. Otherwise, we likely need to add some override rules here: https://github.com/Shopify/polaris/blob/main/tsconfig.eslint.json

Reply:
I feels weird adding override rules for moving dependencies to the root workspace. I wonder if there is a way typescript can understand the monorepo -_-

Reply:
```js
// .eslintrc.js
{
  overrides: [
    {
      files: ['**/rollup.config.js'],
      rules: {
        'import/no-extraneous-dependencies': 'off',
      }
    }
  ]
}
```

Reply:
Or actually even better reading the import/no-extraneous-dependencies doc: https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-extraneous-dependencies.md#options suggests there's a `packageDir` option where you can teach it about multiple package.json files (ie. the one at the root and the ones on a per-package basis)

Reply:
OOoo thanks Ben and Aaron. Let me try the `packageDir` approach!